entity-framework/core/miscellaneous/testing/testing-sample.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: EF Core testing sample - EF Core
-description: Sample showing how to test applications that use EF Core
+description: Sample showing how to test applications that use Entity Framework Core
```

Comment:
applications that use - mismatch in cardinality.

Reply:
Are you saying 'that' is inappropriate here because applications is plural? Am definitely not aware of this rule and interested in learning more :)

Are you saying it should be 'which' instead? Time to reconfirm my knowledge of restrictive vs. non-restrictive relative pronouns...

Reply:
```suggestion
description: Sample showing how to test applications, that use Entity Framework Core
```
How about above? I see that "that" is being used to refer to Sample which is singular so it is correct in that sense but context made it ambiguous for me.

Reply:
Ah no, the idea here was for "that" to refer to the applications (which use EF), not to the sample (restrictive relative pronoun, i.e. telling us which applications are being discussed). I think the suggested comma doesn't work for that, does it make it better if I change "that" to "which"?

Reply:
If it refers to applications then use `those` or `which`.

Reply:
Changed to which